src/routes/create-entry-point-route.ts

```diff
@@ -80,8 +86,26 @@ export function createEntryPointRoute<
     };
   }
 
+  // This is needed to avoid cases where the query has been disposed of but the
+  // router would not normally revalidate and rerun the loader which is needed
+  // to reload the query.
+  // See https://github.com/loop-payments/react-router-relay/issues/15.
```

Comment:
I'm not sure I follow this part. My (definitely limited) understanding of `shouldRevalidate` is that it can be used to prevent calling the loader when it would've normally been called, but it can't trigger a load that wouldn't have otherwise happened?